packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -243,7 +244,6 @@ class UpgradeCommandRunner {
   /// If the user is on a deprecated channel, attempts to migrate them off of
   /// it.
   Future<void> upgradeChannel(FlutterVersion flutterVersion) async {
-    globals.printStatus('Upgrading Flutter from $workingDirectory...');
```

Comment:
I would leave this message in upgradeChannel, and instead pass the upstreamRevision revision in as an argument.

Reply:
Understood. I will modify it to pass in arguments.